test/lint/streams.rb

```diff
@@ -89,6 +89,16 @@ def test_xadd_with_maxlen_and_approximate_option
       assert_match ENTRY_ID_FORMAT, actual
     end
 
+    def test_xadd_with_minid_and_approximate_option
+      omit_version('6.2.0')
```

Comment:
With newer commands and argument, we have to specify which version introduced them.